lib/app.js

```diff
@@ -11,6 +11,7 @@ const unleashSession = require('./middleware/session');
 const responseTime = require('./middleware/response-time');
 const requestLogger = require('./middleware/request-logger');
 const validator = require('./middleware/validator');
+const authBuiltin = require('./auth-builtin');
```

Comment:
Not sure I feel the name is awesoem, if we stick with it though - it should be `built-in`

Reply:
Alternatives `simple-session` / `simple-cookie`  or something.....

Reply:
agreed, we need a better name!

Reply:
how about `unsecure-session`  or `unsecure-authentication`?